app/banner.test.js

```diff
@@ -44,22 +44,4 @@ describe('Banner', () => {
       done(err)
     })
   })
-
-  it.skip('should be dismissable', done => {
```

Comment:
This test is not really that useful since we'd notice if the button didnt work anymore, and if it broke it would just impact us on the dev team and would be a minor annoyance.